adagrams/game.py

```diff
@@ -1,11 +1,120 @@
+import random
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+    ('A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'): 1,
+    ('D', 'G'): 2,
+    ('B', 'C', 'M', 'P'): 3,
+    ('F', 'H', 'V', 'W', 'Y'): 4,
+    ('K',): 5,
+    ('J', 'X'): 8,
+    ('Q', 'Z'): 10
+}
+
+
+def create_letter_pool_list():
+    '''returns a total list of letters from LetterPool'''
+    letter_pool_list = [letter for letter, freq in LETTER_POOL.items() for _ in range(freq)]
+    return letter_pool_list 
+
 def draw_letters():
-    pass
+    '''returns a list of 10 random letters from LetterPool'''
+    letter_pool_list = create_letter_pool_list()
+    letter_bank = []
+
+    for _ in range(10): 
+        random_num = random.randint(0, len(letter_pool_list) -1)
+        letter = letter_pool_list[random_num] 
+        letter_bank.append(letter)
+        letter_pool_list.pop(random_num)      
+    return letter_bank
+
+def count_letters(list_or_str, letter):
+    '''returns the count of a letter in a str or list'''
+    count = 0
+    for item in list_or_str:
+        if item == letter:
+            count += 1
+    return count
 
 def uses_available_letters(word, letter_bank):
-    pass
+    '''returns true if the word uses available letters'''
+    word = word.upper()
```

Comment:
Nice work on ensuring that all the letters where of the same casing. You could also use `casefold`, which is used for caseless matching. You can find more information on the method [here](https://docs.python.org/3/library/stdtypes.html#str.casefold).

Also be mindful of re-assigning the parameter of a function. It can be a side effect of your function if the caller doesn't expect for the data that they passed in to be altered.